### ISSUE TITLE
pem or p12/cer?

### ISSUE THREAD

**Issue by Luccifer** [alexeyxo/swift-apns#2]
Hello! Wanna use it in backend, Perfect server on swift f.ex. Than I have plans to set up it on linux, so need I a pem format of leave it p12?

**Comment by alexeyxo** [comment]
The Library supports "p12" certificates. Maybe, I will be do support of pem/cer.
But I think use "p12" it easy.

**Comment by Luccifer** [comment]
Yep, but I saw a lot of tutorials where on Linux it usually uses pem-format... Just for the future :)
